Makefile

```diff
@@ -95,6 +95,7 @@ npm:
 
 ## clean: Clear the builds and temporary files.
 clean:
+	rm -rf shared/data/bundled/tracker-lookup.json
```

Comment:
If we're committing this to source control, why are we deleting on clean?

Either this is source control, then we don't need these makefile changes. Then we update it with `npm run bundle-config`.

Alternatively, it's not in source control, and generated with the build and not by `bundle-config`.

Reply:
As I was getting the following with Make without it when running `npm run bundle-confiug`:
```

> bundle-config
> node scripts/bundleConfig.mjs && make shared/data/bundled/tracker-lookup.json

Config imported successfully
make: `shared/data/bundled/tracker-lookup.json' is up to date.
```